wlauto/utils/uboot.py

```diff
@@ -114,3 +120,14 @@ def empty_buffer(self):
             pass
         self.conn.buffer = ''
 
+    def _set_new_line_sep(self):
+        uboot_text = self.conn.before
+        uboot_ver = self.uboot_regex.findall(uboot_text)
+
+        if uboot_ver:
+            year, month = uboot_ver[0].split(".")
```

Comment:
This is brittle -- assumes a very specific version format. use `distutils.version.LooseVersion` for comparison instead.